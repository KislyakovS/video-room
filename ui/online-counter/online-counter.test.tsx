import { render } from '@testing-library/react';

import OnlineCounter from './online-counter';

describe('<OnlineCounter>', () => {
    it('snapshot', () => {
        const { container } = render(<OnlineCounter count={10} />);

        expect(container).toMatchSnapshot();
    });
});