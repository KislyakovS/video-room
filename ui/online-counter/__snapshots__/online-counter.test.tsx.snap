// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`OnlineCounter Component: snapshot 1`] = `
<div>
  <div
    class="wrapper"
  >
    <i
      class="rec"
    />
    <span>
      Online 
      10
    </span>
  </div>
</div>
`;
